fakebook/src/app/services/notifications.service.ts

```diff
@@ -20,10 +20,10 @@ export class NotificationsService {
   >();
   private hubConnection: HubConnection;
   public notificationsObs = this.notifications.asObservable();
-  private token = this.authService.oktaAuth.getAccessToken();
+  private token = this.auth.authorizationHeaderValue.split("Bearer")[1].trimLeft();
```

Comment:
it'd be better to expose a different property from the auth service, especially since that service in theory supports more than just bearer tokens. but i won't require a change